Resources/Prototypes/Roles/Antags/nukeops.yml

```
@@ -131,7 +131,7 @@
     #outerClothing: ClothingOuterHardsuitSyndieMedicBiocode # Sunrise-Edit
     #shoes: ClothingShoesBootsMagSyndie # Sunrise-Edit
     id: SyndiAgentPDA
-    belt: ClothingBeltMilitaryWebbingMedFilled # Sunrise-Edit
+    belt: ClothingBeltMilitaryWebbingMedSunriseFilled # Sunrise-Edit
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)